Dockerfile

```diff
@@ -1,17 +1,15 @@
 # Build the authorino binary
-FROM golang:1.17 as builder
-
+FROM registry.access.redhat.com/ubi8/go-toolset:1.17.10 as builder
```

Comment:
might want to use `1.17` floating tag?

Reply:
Same conversation going on here: https://github.com/Kuadrant/authorino-operator/pull/85#discussion_r930731542